catalog-info.yaml

```diff
@@ -0,0 +1,16 @@
+# https://backstage.io/docs/features/software-catalog/
+# https://backstage.io/docs/features/software-catalog/descriptor-format
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: pantheon-content-cloud-sdk
+  description: An SDK to provide a comprehensive set of APIs and tools to facilitate the development of a pantheon site using various supported starters
```

Comment:
"... of a Pantheon Content Cloud site" perhaps? Could be Content Publisher too if you so wish.

Reply:
gt U, added the change.